test/support/generator/get-company-generator.ts

```diff
@@ -228,6 +229,7 @@ export class GetCompanyGenerator extends AbstractGenerator<CompanyValues, Genera
         country: v.country,
       },
       industries,
+      isActive: v.isActive,
```

Comment:
Will `isActive` always be providedm wondering if this should be optional chained?